multisrc/overrides/madara/fusionscanlation/src/FusionScanlation.kt

```diff
@@ -1,13 +1,17 @@
 package eu.kanade.tachiyomi.extension.es.fusionscanlation
 
-import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.multisrc.madara.Madara
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
 import okhttp3.OkHttpClient
 import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class FusionScanlation : MangaThemesia("Fusion Scanlation", "https://fusionscanlation.com", "es", dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale("es"))) {
+class FusionScanlation : Madara("Fusion Scanlation", "https://fusionscanlation.com", "es", SimpleDateFormat("d 'de' MMMM 'de' yyyy", Locale("es"))) {
+
```

Comment:
Please bump the `versionId` so users will be forced to migrate to update the URLs.

```suggestion

override val versionId = 2
```